pkg/sql/sql.y

```diff
@@ -215,6 +215,11 @@ select
 }
 ;
 
+opt_semicolon
```

Comment:
Let us don't mix shortened words, like `opt`, with non-shortened words, like semicolon, in naming.

Also, by the purpose, it seems that `opt_semicolon` should have the name `end_of_statement`?